File: backend/Reimbursement-Calculation-App/src/main/java/pl/Alski/entity/user/LimitsConfiguration.java
package pl.Alski.entity.user;

public class LimitsConfiguration {

    private double dailyAllowanceRate;
    private double carMileageRate;
    private int totalReimbursementLimit;
    private int mileageLimitInKilometers;


    //think whether configuration should be global, or user-deppendent?
    //think how to solve calculation result per receipt type, how to store

    //think how to introduce multiple receipt types, each should have own limit
}

File: frontend/ReimbursementApp/src/app/common/limits-configuration.ts
export class LimitsConfiguration {
    dailyAllowanceRate: number = 0;
    carMileageRate: number = 0;
    totalReimbursementLimit: number = 0;
    mileageLimitInKilometers: number = 0;
    receiptLimits: { [key: string]: number } = {};
}
